### ISSUE TITLE
Can't remap tab change

### ISSUE THREAD

**Issue by hkupty** [neovim/neovim#2365]
For some time now I cannot bind window change commands, such as `<C-w> j` to `<C-j>`.

None of the below would work:
``` vim
map <C-j> <C-w>j
nmap <C-j> <C-w>j
nnoremap <C-j> <C-w>j
map <c-j> <c-w>j
nmap <c-j> <c-w>j
nnoremap <c-j> <c-w>j
map <c-j> <C-w>j
nmap <c-j> <C-w>j
nnoremap <c-j> <C-w>j
map <C-j> <c-w>j
nmap <C-j> <c-w>j
nnoremap <C-j> <c-w>j
```
Hopefully I'm doing something wrong.. :smile:

**Comment by hkupty** [comment]
Well, I was making some tests and it looks like its related to `tmux`and to https://github.com/christoomey/vim-tmux-navigator.. I'll keep on checking, if it turns out to be a defect with the plugin, I'll close the issue..

**Comment by hkupty** [comment]
Yep.. Plugin stopped working for some reason after TUI. Should I close this issue?

**Comment by splinterofchaos** [comment]
You mean after the TUI got merged? Sounds like a regression so the issue should stay open, but a little more detail is necessary. If the problem can be replicated without the plugin, or it can be narrowed down to a specific function or part of the plugin, that'd help.

**Comment by hkupty** [comment]
Well, I tested some possibilities and here's what I got:
The issue happens when I'm using tmux. On tmux I have `<C-[hjkl]>` mapped to the respective panel switches.

I'm using the configuration as follows on the plugin README, so it should pass the keys correctly either to neovim, to vim or to tmux to handle.

I tried to manually rebind the keys as stated above, but looks like the plugin does this for me, so I took off that configuration.

**Comment by hkupty** [comment]
If needed, my configuration files are at https://github.com/hkupty/dotfiles/.

Please note that I start neovim as an aliased `vim`, which is exactly what tmux shows me as the running program for the panel.

**Comment by splinterofchaos** [comment]
>  On tmux I have <C-[hjkl]> mapped to the respective panel switches.

Doesn't that mean those key presses will never reach nvim? It just gives me pause that you say before the TUI, this didn't happen.

**Comment by hkupty** [comment]
Yep, it didn't. Actually they will reach nvim because of tmux's config. If the active pane is (n)vim, send the keys, else handle.

**Comment by hkupty** [comment]
Here is the config section that allows tmux to pass the bindings to vim:
```
is_vim='echo "#{pane_current_command}" | grep -iqE "(^|\/)g?(view|n?vim?)(diff)?$"'
bind -n C-h if-shell "$is_vim" "send-keys C-h" "select-pane -L"
bind -n C-j if-shell "$is_vim" "send-keys C-j" "select-pane -D"
bind -n C-k if-shell "$is_vim" "send-keys C-k" "select-pane -U"
bind -n C-l if-shell "$is_vim" "send-keys C-l" "select-pane -R"
bind -n C-\ if-shell "$is_vim" "send-keys C-\\" "select-pane -l"
```

**Comment by hkupty** [comment]
Ok, I just arrived at work and tried some more tests..

It looks like I had a configuration problem on my `.tmux.conf` and now I'm able navigate  with `<C-[jkl]>`, but not with `<C-h>`. Sorry for the noise. As far as I could see, this is not a plugins problem, since the behavior is the same if I disable the plugin.

Also, I switched the bind line on my tmux to send `:echo "test"<CR>` everytime I press `<C-h>` and it sends correctly the string.

So the current situation is that my `.tmux.conf` is exactly the same as I send above. Outside of tmux I can move normally but while on tmux `<C-h>` does not work, even if the keys are sent correctly from tmux to nvim.
